pkg/build/controller/controller.go

```diff
@@ -125,29 +125,8 @@ func (bc *BuildController) nextBuildPhase(build *buildapi.Build) error {
 	// Set the build phase, which will be persisted if no error occurs.
 	build.Status.Phase = buildapi.BuildPhasePending
 
-	// Make a copy to avoid mutating the build from this point on.
```

Comment:
Why is it ok to mutate the build now? In general, an explanation is helpful when something we previously commented explicitly about is undone.